cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -154,16 +156,24 @@ def test_provision_cloud_init_payload(appliance, request, setup_provider, provid
 
     ci_payload = {
         'root_password': 'mysecret',
-        'address_mode': 'Static',
+        'address_mode': 'DHCP',
         'hostname': 'cimachine',
-        'ip_address': '169.254.0.1',
-        'subnet_mask': '29',
-        'gateway': '169.254.0.2',
-        'dns_servers': '169.254.0.3',
-        'dns_suffixes': 'virt.lab.example.com',
         'custom_template': {'name': 'oVirt cloud-init'}
     }
 
+    THERE_IS_A_WAY_TO_CHECK_STATIC_IP_VM = False
+    # TODO on review: How? Perhaps we can check the logs from the cloud-init somehow?
```

Comment:
Discussed a bit in comments how we can validate this when we're setting a link-local address in cloud-init, that will make the VM unreachable from the outside.

Options included:
1. Use the host as a jump box, to reach an address in the internal RHV CIDR range selected for cloud-init's payload
2. Use multiple vNIC devices on the template/VM, which can be added with wrapanapi. Let cloud-init configure the first device, let RHV DHCP assign on the secondary device.

Reply:
I used the jumpbox approach